firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-# Unreleased
+# Unreleased (21.1.0)
+- [changed] With disk persistence enabled, query processing time for most
+  queries is now mainly dependent on the size of the result set rather than the
+  size of the locally cached collection. Queries that use the `limit()` API as
+  well as queries that have not been recently executed may not benefit from
+  this change.
+  If you are updating to v21.1.0, only queries that have been synchronized with
```

Comment:
I don't think you need "If you are updating to v21.1.0".  Even for a brand-new user, the first time running the query will always be the slow path.

Reply:
To me, this was part of "have not been recently executed".  I did end up combining it with the previous list of caveats.